<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>initialconPres</class>
 <widget class="QFrame" name="initialconPres">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>996</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="Lbl_H1">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;INITIAL CONDITIONS (PRESSURE)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="Lbl_Notice">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Initial conditions are already prescribed through 
the shallow water solvers</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="HLY_Pres" stretch="0,0">
     <item>
      <widget class="QLabel" name="Lbl_Pres">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Pressure</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="DSpBx_Pres"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="Lbl_Regions">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Regions&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="Tbl_IniCondTable">
     <property name="toolTip">
      <string>Regions for initial condition</string>
     </property>
     <property name="toolTipDuration">
      <number>10000</number>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>P1(x,y,z)</string>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>P2(x,y,z)</string>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="HLY_AddRegion">
     <item>
      <widget class="QToolButton" name="Btn_AddRegion">
       <property name="toolTip">
        <string>Add regions for defining initial conditions</string>
       </property>
       <property name="toolTipDuration">
        <number>10000</number>
       </property>
       <property name="text">
        <string>Add region</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="Btn_RemRegion">
       <property name="toolTip">
        <string>Remove regions for defining initial conditions</string>
       </property>
       <property name="toolTipDuration">
        <number>10000</number>
       </property>
       <property name="text">
        <string>Remove region</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="VSp">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>404</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
